internal/common/conversion/model_generation.go

```diff
@@ -6,17 +6,18 @@ import (
 )
 
 // CopyModel creates a new struct with the same values as the source struct. Fields in destination struct that are not in source are left with zero value.
-func CopyModel[T any](src any) (*T, error) {
+// It panics if there are some structural problems so it should only happen during development.
+func CopyModel[T any](src any) *T {
```

Comment:
Why not keep the old error returner and support both with error or panic?
But maybe there are no use cases?

Reply:
i didn't find any use case where it would return error, apart from misconfiguration